cEP-0024.md

```diff
@@ -0,0 +1,255 @@
+then auto assign the author of the PR to the mentioned unassigned issues. If
+the author of the PR isn't eligible to get assigned to the mentioned issues,
+missing eligibility conditions will be specified as a comment, which isn't
+configurable.
+
+If someone else is assigned to the mentioned issue, GitMate will
+comment (configurable) on the PR.
+
+If the author of a PR is not a member of the organization, then he/she can't be
+assigned to any of the issues, a label (configurable) will be added on the PR
+so that maintainers can identify these type of PRs and GitMate will comment
+on the PR informing the author of the PR to get invited to the organization
+before working on any issue. Maintainers need to remove the label added by
+GitMate manually.
+
+coala can configure this plugin as:
+
+**Comment message when someone else is assigned to mentioned issues:**
+(use `{issues}` as the placeholder to get links of all assigned issues)
+`Sorry, but the issues {issues} are assigned to someone else. Please try
+working on some other issues.`
+
+**Label to represent that the author of the PR is not a member of the org:**
+`author/non-member`
+
+### Auto-reject PR opened on certain branch names
+
+Normally, contributors should make changes in a different branch from the
+master branch of the forked repository, and submit a PR. But, often coala
+receives PRs from the master branch of the forked repository.
+
+coala needs a mechanism such that when a PR is opened or updated, GitMate
+automatically closes PRs opened on certain branch names (master), adds certain
+label to the PR so that maintainers can identify these type of PRs and comments
+on the PR informing the author of the PR label to work on a different branch.
+Comment message, branch names, the label will be configurable.
+
+coala can configure this plugin as:
+
+**Branch names:** `master`
+
+**Label:** `status/rejected`
+
+**Comment message:** `This PR is rejected. PR should never be opened in master
+branch, work on a different branch other than master and open a new PR, follow
+[tutorial](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html#step-
+3-creating-a-fork-and-testing-your-changes) if
+you repeat this again, you may get banned from coala.`
+
+### Auto-mark PR with necessary labels as per the reviews done
+
+When a reviewer reviews a PR, he/she either approves changes or request changes
+and leaves it there, but the PR would be still in the review queue. So, coala
+needs a better mechanism to clear review queue when any maintainer reviews PR
+or when there is any CI failure.
+
+GitMate will consider only maintainer reviews, to consider developer reviews,
+GitMate has use teams, but using teams in GitMate is not a part of my project.
+GitMate may consider using teams in the future.
+
+When a PR is opened, updated or when any maintainer reviews the PR, GitMate
+will automatically mark labels, as per the following conditions:
```

Comment:
"automatically mark labels" isnt clear.

I guess "add labels" ... ?

Also should say that it will remove the "pending review" label, which still isnt mentioned anywhere in this section